application/app/Models/Task.php

```diff
@@ -136,4 +136,17 @@ public function task_comments()
     {
         return $this->hasMany(TaskComment::class);
     }
+
+    /**
+     * 課題が所有する添付画像を取得.
+     */
+    public function task_files()
+    {
+        return $this->hasMany(TaskFile::class);
+    }
+
+    public function count_files()
```

Comment:
くどいですが、下記のほうが良いです。
（他のファイルが増える可能性もありますし、あまり抽象度の高い名前にしないほうがベターです）
```suggestion
    public function count_task_files()
```